packages/backend/server/src/modules/users/resolver.ts

```diff
@@ -220,27 +210,55 @@ export class UserResolver {
       ttl: 60,
     },
   })
-  @Mutation(() => AddToNewFeaturesWaitingList)
-  async addToNewFeaturesWaitingList(
-    @CurrentUser() user: UserType,
-    @Args('type', {
-      type: () => NewFeaturesKind,
-    })
-    type: NewFeaturesKind,
+  @Mutation(() => Int)
+  async addToEarlyAccess(
+    @CurrentUser() currentUser: UserType,
     @Args('email') email: string
-  ): Promise<AddToNewFeaturesWaitingList> {
-    if (!isStaff(user.email)) {
+  ): Promise<number> {
+    if (!this.feature.isStaff(currentUser.email)) {
       throw new ForbiddenException('You are not allowed to do this');
     }
-    await this.prisma.newFeaturesWaitingList.create({
-      data: {
-        email,
-        type,
-      },
-    });
-    return {
-      email,
-      type,
-    };
+    const user = await this.users.findUserByEmail(email);
+    if (user) {
+      return this.feature.addEarlyAccess(user.id);
+    } else {
+      const user = await this.auth.createAnonymousUser(email);
+      return this.feature.addEarlyAccess(user.id);
+    }
+  }
+
+  @Throttle({
+    default: {
+      limit: 10,
+      ttl: 60,
+    },
+  })
+  @Mutation(() => Int)
+  async removeEarlyAccess(
+    @CurrentUser() currentUser: UserType,
+    @Args('email') email: string
+  ): Promise<number> {
+    if (!this.feature.isStaff(currentUser.email)) {
+      throw new ForbiddenException('You are not allowed to do this');
+    }
+    const user = await this.users.findUserByEmail(email);
+    if (!user) {
+      throw new BadRequestException(`User ${email} not found`);
+    }
+    return this.feature.removeEarlyAccess(user.id);
+  }
+
+  @Throttle({
+    default: {
+      limit: 10,
+      ttl: 60,
+    },
+  })
+  @Query(() => [UserType])
+  async listEarlyAccess(@CurrentUser() user: UserType): Promise<UserType[]> {
+    if (!this.feature.isStaff(user.email)) {
```

Comment:
seems we need sys-admin accounts now, let's add a todo

Reply:
Considering whether we need to implement a feature-based implementation to allow specific accounts to have the ability to adjust features, or use a separate user pool to provide backend management capabilities—in our scenario, it seems a bit strange to include the management background in affine.
Maybe we should discuss this problem when removing `next-auth`